pkg/pillar/go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
 	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f
 	google.golang.org/grpc v1.47.0
-	google.golang.org/protobuf v1.28.0
+	google.golang.org/protobuf v1.28.1
```

Comment:
Are we required to update the protobuf version for this patch?

Reply:
actually no, updated the PR.